### PR TITLE
Several enhancements: viewer functionality (on the forms), send only …

### PR DESCRIPTION
**Pull request by yanjoseph**
…feedback forms over specific associates of a specific direct manager (active gmail session), custom signature

### CODE REVIEW

File: Code.gs [hunk 1]
```diff
@@ -57,10 +65,14 @@ function GenerateForms() {
...
       associateForm.addEditor(managerEmail);
+      if (viewerEmail != managerEmail && viewerEmail != "") {
+        //associateForm.addViewer(viewerEmail); // viewers aren't supported
+        associateForm.addEditor(viewerEmail);
+      }
 
       // send email to reviewers and cc manager
       for (var j = 0; j < reviewers.length; j++) {
-        email = GmailApp.createDraft(reviewers[j], title, "You are kindly asked to provide feedback via this form.\n\n" + associateForm.getPublishedUrl() + "\n\nResults will be visible only to the direct lead of the reviewed person.\n\n-- \nFederico\n", { cc: managerEmail });
+        email = GmailApp.createDraft(reviewers[j], title, "You are kindly asked to provide feedback via this form.\n\n" + associateForm.getPublishedUrl() + "\n\nResults will be visible only to the direct lead of the reviewed person.\n\n" + emailSignature, { cc: managerEmail });        //row.getCell(1,10).setValue("Email Draft");
```
Comment (simon3z) [inline Code.gs]: > //row.getCell(1,10).setValue("Email Draft");

@yanjoseph seems something unintended? Should you drop it?

Comment (yanjoseph) [reply Code.gs]: removed the redundant comment

**Comment by simon3z** [comment]
@scollier can you review?

**Comment by scollier** [comment]
Can we get some generic instructions added to the repo for how to test this, what the expected results are, etc...? @yanjoseph

**Comment by simon3z** [comment]
> Can we get some generic instructions added to the repo for how to test this, what the expected results are, etc...?

@yanjoseph offered to help with these features, maybe documentation is something we can all contribute to in a different PR?

**Review by scollier — APPROVED** [review]
LGTM. I was able to run the code and generate the forms. I will work on a doc PR with the steps required.

/cc @simon3z

**Pull request merged**